### DISCUSSION TITLE
Is it possible to send sext sms anonymously?

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#3084] [Q&A] [answered]
Is it possible to change `self::id` to the id of any random player or not display it at all?

**Comment by Rxann** [comment, marked as answer]
I could be wrong but I think sending sms anonymously was patched a while back.

**Reply by GreySideMM** [reply]
I'm not sure but since sext seems to be a script event and this note [right here](https://github.com/YimMenu/YimMenu/blob/54c7dada828384b96c4bffd3a0d8fd87c98a017d/src/views/debug/view_debug_script_events.cpp#L58) links (missing) self::id to AC detection.. this might be no good idea anyway. But also no glue about the AC and how up-to-date this note might be.
